src/bio/bio_xstream.c

```diff
@@ -124,11 +201,19 @@ populate_whitelist(struct spdk_env_opts *opts)
 	const char			*val;
 	size_t				 i;
 	int				 rc = 0;
+	bool				 vmd_enabled = false;
 
 	/* Don't need to pass whitelist for non-NVMe devices */
 	if (nvme_glb.bd_bdev_class != BDEV_CLASS_NVME)
 		return 0;
 
+	/*
+	 * Optionally VMD devices will be used, and will require a different
+	 * transport id to pass to whitelist for DPDK.
+	 */
+	if (spdk_conf_find_section(NULL, "Vmd") != NULL)
+		vmd_enabled = true;
```

Comment:
yes . the section only gets written if vmd is present

Reply:
right, the check to see if the section is "enabled" with spdk_conf_section_get_boolval() is not needed since the VMD section in the config will not be present unless VMD is present and enabled, and the only value of VMD section is "Enable True"